### PR TITLE
Js cleanup named pipes

### PR DESCRIPTION
**Pull request by jsingle**
@paetling PR does:
 * Remove `wait` on pid, which was causing to block on background threads. 
 * Don't send output of `always` to a log file.  I don't think we need to do this since that's the main command that gets run on the container (just use dusty logs)

### CODE REVIEW

File: dusty/command_file.py [hunk 1]
```diff
@@ -22,11 +22,8 @@ def _tee_output_commands(command_to_tee):
         'PIPEFILE={}_pipe_file'.format(tee_function_name),
         'rm -f $PIPEFILE',
         'mkfifo $PIPEFILE',
-        'tee {}.log < $PIPEFILE &'.format(os.path.join(constants.CONTAINER_LOG_PATH, command_to_tee)),
-        'TEEPID=$!',
+        '(tee {}.log < $PIPEFILE || true; rm -f $PIPEFILE) &'.format(os.path.join(constants.CONTAINER_LOG_PATH, command_to_tee)),
```
Comment (paetling) [inline dusty/command_file.py]: why do you need the || true?  wont the result of this statement just be the command after the ;?

**Comment by paetling** [comment]
I do not really understand your argument for the second part. Can't you also look at logs for once?  I guess since it does not always run it is not worth saving?

File: dusty/command_file.py (hunk 1 above)
Comment (jsingle) [inline dusty/command_file.py]: nope because we `set -e`, which anecdotally persists to this sub shell thingy

**Comment by jsingle** [comment]
My argument is that `once` output is worth saving.  `always` output is not

File: dusty/command_file.py (hunk 1 above)
Comment (paetling) [inline dusty/command_file.py]: But if outputting to the log fails, don't we want the whole thing to fail?

Comment (jsingle) [inline dusty/command_file.py]: It won't anyway because it's run in the background.  And that'd debateable

**Comment by paetling** [comment]
I like the once always distinction. I think we mention in our logs somewhere about them. Worth a quick glance and correction if it is there.

File: dusty/command_file.py (hunk 1 above)
Comment (paetling) [inline dusty/command_file.py]: i see.   ok.

**Comment by paetling** [comment]
just the logs thing and then :ghost: :cow:

**Comment by jsingle** [comment]
Closes https://github.com/gamechanger/dusty/issues/380

**Pull request merged**